src/style/variables.scss

```diff
@@ -0,0 +1,16 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@400;600&display=swap');
+
+// Colors <---------
+
+$mainBackground: #C9D2Df;
```

Comment:
Loving this color palette!

Reply:
I'm excited to see how it translates into our interface!

Reply:
Can't wait to see how this looks!!! Also, would love insight on the process of making a color palette @jkwest-93